app/api/organizations/[orgId]/sso/route.ts

```
@@ -35,8 +35,11 @@ const PatchBodySchema = z
     // = "OWNER"` would make the first SSO user co-owner. See audit
     // Phase A.1 + docs/enterprise/20-iam-and-security/01-sso-and-authentication.md.
     defaultRoleForAutoJoin: JitDefaultRoleSchema.optional(),
+    // Optimistic-lock clock. When present, a concurrent admin PATCH that
+    // already bumped the version 409s instead of silently clobbering.
+    expectedVersion: z.coerce.number().int().min(1).optional(),
   })
-  .refine((v) => Object.keys(v).length > 0, {
+  .refine((v) => Object.keys(v).filter((k) => k !== "expectedVersion").length > 0, {
```

ESLint

[error] Use Array.some() instead of Array.find() and Array.filter().length checks when checking for element existence

(e18e/prefer-array-some)